crl-janitor/sa.tf

```diff
@@ -54,7 +61,7 @@ resource "google_folder_iam_member" "app_folder_roles" {
 
   provider = google.target
   folder  = var.google_folder_id
-  role     = "roles/editor"
+  role     = local.app_folder_roles[count.index]
```

Comment:
need to modify the count on line 60
`count = var.enable && (var.google_folder_id != "") ? length(local.app_folder_roles) : 0`

Reply:
ah, right, thanks!